proposed/http-factory/http-factory.md

```diff
@@ -163,14 +170,17 @@ fwrite($resource, $body);
 Has the ability to create streams for uploaded files.
 
 ```php
-namespace Psr\Http\Message;
+namespace Psr\Http\Factory;
+
+use Psr\Http\Message\UploadedFileInterface;
 
 interface UploadedFileFactoryInterface
 {
     /**
      * Create a new uploaded file.
      *
-     * If a string is passed it is assumed to be a file path.
+     * If a string is used to create the file, a temporary resource will be
+     * created with the content of the string.
```

Comment:
Honestly, I wonder if this interface is needed at all. What's the use case for creating an `UploadedFile` that isn't already provided by `ServerRequest`? It's not possible to use `UploadedFile` for transmitting files (client usage) and PSR-7 requires that `ServerRequest` provide functionality to get uploaded files.

I would certainly be happier if this was dropped from PSR-17, as I have found zero real-world use for it.